cpp/CMakePresets.json

```diff
@@ -395,6 +421,16 @@
       "displayName": "Release build for PyArrow with everything enabled",
       "cacheVariables": {}
     },
+    {
+      "name": "ninja-release-emscripten-python",
+      "inherits": [
+        "emscripten-overrides",
+        "base-release",
+        "features-python"
+      ],
+      "displayName": "Release build which builds an emscripten library, plus PyArrow for Pyodide",
```

Comment:
Ah, we don't include any PyArrow related features in Arrow C++ now. (GH-14117)
`features-python` is just for enabling Arrow C++ features that are used from PyArrow.